genomepy/functions.py

```diff
@@ -290,21 +307,24 @@ def generate_env(fname=None):
                 fout.write("{}\n".format(env))
 
 
-def glob_fa_files(dirname):
+def glob_fa_files(dirname, ext='fa'):
```

Comment:
Update function name, as it is now not fa-specific. `glob_ext_files()`?

Reply:
makes sense!